obc_transceiver_simulator/packets.py

```diff
@@ -56,6 +58,10 @@ def receive_rx_packet():
 
         # Need 2 <CR> bytes to start/end message
         if len(cr_indices) >= 2:
+            # Drop packet?
+            if (random.uniform(0,1) < Global.downlink_drop):
+                print("Downlink Packet Dropped")
+                return None
```

Comment:
It's the right logic but if you do it at this point in the code, you don't know if it actually detected a packet with the correct format - should move this to line 82.